### PR TITLE
Bump to v1.19.1

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
As the title says.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@nalind @rhatdan PTAL

File: contrib/rpm/buildah.spec [hunk 1]
```diff
@@ -100,6 +100,9 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 %{_datadir}/bash-completion/completions/*
 
 %changelog
+* Fri Jan 8, 2021 Tom Sweeney <tsweeney@redhat.com> 1.19.1-1
```
Comment (nalind) [inline contrib/rpm/buildah.spec:103]: Date looks off, but not a blocker.

Comment (TomSweeneyRedHat) [reply contrib/rpm/buildah.spec]: oops!  Thanks, fixed and repushed.

**Comment by nalind** [comment]
LGTM, but it's either this one and moving the v1.19.1 tag, or #2911.

**Comment by TomSweeneyRedHat** [comment]
I think it needs to be this one as #2011 has a different version of c/common than what is in Podman 3.0 and they need to sync.

**Comment by rhatdan** [comment]
Already forced this through, but we need another fix.

**Comment by nalind** [comment]
> Already forced this through, but we need another fix.

Can you elaborate?  I'm looking at this and I have _no_ idea what's going on.

**Comment by rhatdan** [comment]
Were all set, I opened PRs to fix an issue with pkg/overlay. 
Buildah 1.19.2 is the current release.

**Pull request closed without merging**